src/olmo_core/train/checkpoint.py

```diff
@@ -128,6 +154,20 @@ def load(
         """
         dir = normalize_path(dir)
 
+        if is_url(dir) and self.pre_download:
+            target = self.work_dir / "load" / os.path.basename(dir)
+            if get_fs_local_rank() == 0:
+                copy_dir(dir, target, save_overwrite=self.save_overwrite)
+            barrier()
```

Comment:
This is means that if the copy takes longer than 10 minutes, we will definitely timeout. Is there a way to extend the timeout, or have the other ranks participate in the copying somehow?

Reply:
In your PR for the 32B config you extended the timeout to 30 minutes. Is that enough?

Reply:
Yeah, but 30 minutes is really bad when you don't need it. Might make a 1B model stall out for 30 minutes when it doesn't need to.

Reply:
actually we can just increase the timeout for the checkpointing PG: d67f767